packages/cli/src/util/deploy/create-deploy.ts

```diff
@@ -19,7 +19,7 @@ export default async function createDeploy(
   cwd?: string
 ): Promise<any | DeploymentError> {
   try {
-    return await now.create(paths, createArgs, org, isSettingUpProject, cwd);
+    return now.create(paths, createArgs, org, isSettingUpProject, cwd);
```

Comment:
We still want `await` here otherwise a thrown error won't be caught in the try/catch here.

Reply:
i feel like the async/await is the wrong pattern here, but i don't think this is a high volume promise call so there is little point trying to optimize. If it makes the error handling easier than no need to change either 😄